packages/alfa-dom/test/h.spec.ts

```diff
@@ -10,16 +10,19 @@ import { Element } from "../src/node/element";
 import { Type } from "../src/node/type";
 
```

Comment:
We should add tests that:
* `serializationID` is correctly propagated if provided.
* `serializationID` is created if not provided (just test presence, of course).